sdk/search/azure-search-documents/README.md

```diff
@@ -113,61 +168,252 @@ SearchAsyncClient searchAsyncClient = new SearchClientBuilder()
 
-There are several types of operations that can be executed against the service:
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L122-L147 -->
+```Java
+// We'll connect to the Azure Cognitive Search public sandbox and send a
+// query to its "nycjobs" index built from a public dataset of available jobs
+// in New York.
+String serviceName = "azs-playground";
+String indexName = "nycjobs";
+String apiKey = "252044BE3886FE4A8E3BAA4F595114BB";
+
+// Create a SearchClient to send queries
+String serviceEndpoint = String.format("https://%s.search.windows.net/", serviceName);
+AzureKeyCredential credential = new AzureKeyCredential(apiKey);
+SearchClient client = new SearchClientBuilder()
+    .endpoint(serviceEndpoint)
+    .credential(credential)
+    .indexName(indexName)
+    .buildClient();
 
--   [Index management operations](https://docs.microsoft.com/en-us/rest/api/searchservice/index-operations). Create, delete, update, or configure a search index.
--   [Document operations](https://docs.microsoft.com/en-us/rest/api/searchservice/document-operations). Add, update, or delete documents in the index, query the index, or look up specific documents by ID.
--   [Indexer operations](https://docs.microsoft.com/en-us/rest/api/searchservice/indexer-operations). Automate aspects of an indexing operation by configuring a data source and an indexer that you can schedule or run on demand. This feature is supported for a limited number of data source types.
--   [Skillset operations](https://docs.microsoft.com/en-us/rest/api/searchservice/skillset-operations). Part of AI workload, a skillset defines a series of a series of enrichment processing steps. A skillset is consumed by an indexer.
--   [Synonym map operations](https://docs.microsoft.com/en-us/rest/api/searchservice/synonym-map-operations). A synonym map is a service-level resource that contains user-defined synonyms. This resource is maintained independently from search indexes. Once uploaded, you can point any searchable field to the synonym map (one per field).
+// Let's get the top 5 jobs related to Microsoft
+SearchPagedIterable searchResultsIterable = client.search("Microsoft", new SearchOptions().setTop(5),
+    Context.NONE);
+for (SearchResult searchResult: searchResultsIterable) {
+    SearchDocument document = searchResult.getDocument(SearchDocument.class);
+    String title = (String) document.get("business_title");
+    String description = (String) document.get("job_description");
+    System.out.println(String.format("The business title is %s, and here is the description: %s",
+        title, description));
+}
+```
+
+## Key concepts
+
+An Azure Cognitive Search service contains one or more indexes that provide
+persistent storage of searchable data in the form of JSON documents.  _(If
+you're brand new to search, you can make a very rough analogy between
+indexes and database tables.)_  The Azure.Search.Documents client library
```

Comment:
Was `Azure.Search.Documents` pulled from .NET? For Java should it be `azure-search-documents`?

Reply:
Yes, changed in latest commits.